kpi/views/kpi_views.xml

```diff
@@ -0,0 +1,154 @@
+                    <filter string="Type" context="{'group_by':'kpi_type'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="view_kpi_form" model="ir.ui.view">
+        <field name="name">kpi.form</field>
+        <field name="model">kpi</field>
+        <field name="arch" type="xml">
+            <form string="Key Performance Indicator">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                            <field name="threshold_id"/>
+                            <field name="category_id"/>
+                        </group>
+                        <group>
+                            <field name="value" colspan="2"/>
+                            <field name="active" colspan="2"/>
+                            <field name="company_id"
+                                   groups="base.group_multi_company"/>
+                            <button name="compute_kpi_value"
+                                    string="Compute KPI Now"
+                                    colspan="2"
+                                    type="object"
+                                    groups="kpi.group_kpi_manager"/>
+                        </group>
+                    </group>
+                    <notebook colspan="6">
+                        <page string="History">
+                            <field name="history_ids" readonly="1" nolabel="1"/>
+                        </page>
+                        <page string="Computation" groups="kpi.group_kpi_manager">
+                            <group col="6">
+                                <field name="periodicity" colspan="3"/>
+                                <field name="periodicity_uom" colspan="3"/>
+                                <field name="next_execution_date" colspan="3"/>
+                                <separator string="KPI Computation" colspan="6"/>
+                                <newline/>
+                                <field name="kpi_type" colspan="2"/>
+                                <field name="dbsource_id" colspan="2"
+                                       attrs="{'invisible' : [('kpi_type', '!=', 'external')]}"/>
+                                <newline/>
+                                <field name="kpi_code" colspan="6"/>
+                            </group>
+                        </page>
+                        <page string="Description">
+                            <field name="description" nolabel="1"/>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="open_kpi_dashboard">
+        <field name="name">KPI Dashboard</field>
+        <field name="res_model">kpi</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">kanban,form</field>
```

Comment:
Just kanban, if you add form, you are may enter to the configuration, but the usual user only needs to access to the kanban